src/TBuffer.cpp

```diff
@@ -779,6 +779,10 @@ TBuffer::TBuffer(Host* pH)
 #endif
 }
 
+auto operator""_MB(unsigned long long const x)
```

Comment:
Nice trick. Could you just add a comment here what this is for?

Reply:
🤯

Reply:
Done.